test/integration/job/job_test.go

```diff
@@ -2117,6 +2139,140 @@ func BenchmarkLargeIndexedJob(b *testing.B) {
 				}
 				b.StartTimer()
 			}
+			b.Logf("Time elapsed for %s: %v", name, b.Elapsed())
```

Comment:
```suggestion
			b.Logf("Time elapsed: %v", name, b.Elapsed())
```

The name should already be included by the framework.

Reply:
Actually, this is not needed. The benchmark measurements are reported even without the log, I just missed the lines. Here is an example: 
```
BenchmarkLargeIndexedJob/regular_indexed_job_without_failures;_size=10-48                       	       1	3034342185 ns/op	14391160 B/op	  164352 allocs/op
BenchmarkLargeIndexedJob/regular_indexed_job_without_failures;_size=100-48                      	       1	3050613253 ns/op	111100464 B/op	 1324757 allocs/op
```